serverless/event_sources/knative-event-sources.adoc

```diff
@@ -5,26 +5,20 @@ include::modules/common-attributes.adoc[]
 :context: knative-event-sources
 toc::[]
 
-An _event source_ is an object that links an event producer with an event _sink_, or consumer. A sink can be a Knative Service, Channel, or Broker that receives events from an event source.
+An _event source_ is an object that links an event producer with an event _sink_, or consumer. A sink can be a Knative service, channel, or broker that receives events from an event source.
+
+[id="knative-event-sources-creating"]
+== Creating event sources
```

Comment:
Since this is more of a conceptual module on the supported event source types, how about renaming this to 'Supported event source types'?
And maybe move the links on L 22 and 23 to Additional Resources.

Reply:
Sure, Ashleigh. Maybe link this comment to a tracker issue you have for KafkaSource? That way it won't be forgotten.

Reply:
Kafka source PR is this one: https://github.com/openshift/openshift-docs/pull/26990
The previous one that merged was for the kafka install